source/admin_manual/list_of_events.rst

```diff
@@ -2602,6 +2602,22 @@ metacache_user_clean_finished
 | error                 | Error message if the upload failed                   |
 +-----------------------+------------------------------------------------------+
 
+metacache_pull_begin
+^^^^^^^^^^^^^^^^^^^^
+metacache_pull_finished
+^^^^^^^^^^^^^^^^^^^^^^^
+
++-----------------------+------------------------------------------------------+
+| Field                 | Description                                          |
++=======================+======================================================+
+| source-backend-host   | From which host metacache is being pulled            |
```

Comment:
Should be underscores. Also I think just source_host would be enough. It might not even be a backend at some point, if we're going to support pulling via a proxy.